client/my-sites/email/gsuite-add-users/index.jsx

```diff
@@ -81,24 +79,16 @@ class GSuiteAddUsers extends React.Component {
 	}
 
 	handleContinue = () => {
-		const { domains, planType, selectedSite } = this.props;
+		const { domains, productType, selectedSite } = this.props;
```

Comment:
Can we not rely on the productSlug that we're already returning in the `domains` object?
```suggestion
		const { domains, selectedSite } = this.props;
		const currentProductSlug = domains[0]?.googleAppsSubscription?.productSlug; // this can definitely be cleaned up!
		const productSlug = currentProductSlug ?? isEnabled( 'google-workspace-migration' ) ?  GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY : GSUITE_BASIC_SLUG;
```